src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1713,4 +1713,16 @@ I agree to all of the foregoing:</value>
   <data name="Error_updating_suppressions_0" xml:space="preserve">
     <value>Error updating suppressions: {0}</value>
   </data>
+  <data name="Inline_Diagnostics_experimental" xml:space="preserve">
+    <value>Inline Diagnostics (experimental)</value>
+  </data>
+  <data name="Display_inline_diagnostics" xml:space="preserve">
+    <value>Display inline diagnostics</value>
+  </data>
+  <data name="Hook_to_end_of_text" xml:space="preserve">
+    <value>Hook to end of text</value>
+  </data>
+  <data name="Hook_to_end_of_window" xml:space="preserve">
+    <value>Hook to end of window</value>
```

Comment:
I don't think I like "Hook" here. Perhaps something like "at the end of the line of code" and "on the right edge of editor window" would work? So it reads like a sentence from "Display inline diagnostics" onwards.

For reference for others:
![image](https://user-images.githubusercontent.com/754264/122124684-fb248400-ce72-11eb-8d77-090ca7ac3115.png)

Reply:
I would also phrase it as: display diagnostics inline

Reply:
fixed

Reply:
![image](https://user-images.githubusercontent.com/40616383/122817804-3f59bf00-d28d-11eb-9033-a1d2f1f78369.png)